tests/integration-tests/configs/common/common.yaml

```diff
@@ -119,14 +119,19 @@ create:
         oss: {{common.OSS_COMMERCIAL_X86}}
         schedulers: ["slurm"]
 createami:
+  test_createami.py::test_invalid_config:
+    dimensions:
+      - regions: ["eu-west-3"]
+        instances: {{ common.INSTANCES_DEFAULT_X86 }}
+        oss: ["alinux2"]
   test_createami.py::test_build_image:
     dimensions:
       - regions: ["eu-west-3"]
         instances: {{ common.INSTANCES_DEFAULT_X86 }}
-        oss: ["alinux2", "ubuntu2004", "centos7"]
+        oss: ["alinux2"] # TODO: add "ubuntu2004" "centos7"
```

Comment:
"ubuntu2004", "centos7" works in this test, and "ubuntu1804" will be fixed by PR https://github.com/aws/aws-parallelcluster/pull/3108